diff -ur ../vips-7.14.3.orig/python/vipsCC/Makefile.am ./python/vipsCC/Makefile.am
--- ../vips-7.14.3.orig/python/vipsCC/Makefile.am	2008-03-07 17:42:34.000000000 -0500
+++ ./python/vipsCC/Makefile.am	2008-05-03 09:53:16.808122270 -0400
@@ -1,4 +1,4 @@
-INCLUDES = -I${top_srcdir}/include @VIPS_CFLAGS@ @VIPS_INCLUDES@ @PYTHON_INCLUDES@
+INCLUDES = -I${top_srcdir}/include @VIPS_CFLAGS@ @VIPS_INCLUDES@ $(PYTHON_INCLUDES)
 
 # we install to a directory inside the python area, since we are a module
 vipsccdir = $(pyexecdir)/vipsCC
diff -ur ../vips-7.14.3.orig/python/vipsCC/Makefile.in ./python/vipsCC/Makefile.in
--- ../vips-7.14.3.orig/python/vipsCC/Makefile.in	2008-04-12 10:37:17.000000000 -0400
+++ ./python/vipsCC/Makefile.in	2008-05-03 09:53:56.824122142 -0400
@@ -304,7 +304,7 @@
 sharedstatedir = @sharedstatedir@
 sysconfdir = @sysconfdir@
 target_alias = @target_alias@
-INCLUDES = -I${top_srcdir}/include @VIPS_CFLAGS@ @VIPS_INCLUDES@ @PYTHON_INCLUDES@
+INCLUDES = -I${top_srcdir}/include @VIPS_CFLAGS@ @VIPS_INCLUDES@ $(PYTHON_INCLUDES)
 
 # we install to a directory inside the python area, since we are a module
 vipsccdir = $(pyexecdir)/vipsCC
Only in ./python/vipsCC: Makefile.in.orig
