--- vips-7.10.16/configure.in.qdist	2005-11-12 12:29:41.000000000 -0500
+++ vips-7.10.16/configure.in	2005-11-25 19:12:10.982474496 -0500
@@ -22,7 +22,7 @@
 # is not backwards compatible?: reset age to 0
 LIBRARY_CURRENT=18
 LIBRARY_REVISION=1
-LIBRARY_AGE=9
+LIBRARY_AGE=8
 
 AM_INIT_AUTOMAKE($PACKAGE,$VERSION)
 
--- vips-7.10.16/configure.qdist	2005-11-12 12:30:01.000000000 -0500
+++ vips-7.10.16/configure	2005-11-25 19:12:31.935289184 -0500
@@ -1599,7 +1599,7 @@
 # is not backwards compatible?: reset age to 0
 LIBRARY_CURRENT=18
 LIBRARY_REVISION=1
-LIBRARY_AGE=9
+LIBRARY_AGE=8
 
 am__api_version="1.9"
 ac_aux_dir=
